src/poliastro/bodies.py

```diff
@@ -168,6 +168,8 @@ class _Earth(_Body):
     J3 = constants.J3_earth
     H0 = constants.H0_earth
     rho0 = constants.rho0_earth
+    critical_inclinations = constants.critical_inclinations_earth
```

Comment:
All attractors have the same critical inclinations and arguments of periapsis, see my other comment.